src/content/en/2024/structured-data.md

```diff
@@ -2,17 +2,611 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTE7UqINKaCzsmdVSUPZhQjlm2Az5T9wcbk4ZMZyJu-Rp8JWXtn9YRfXnAajUqQYLTCnSA2O5FEP6YF/pubchart?oid=778075778&format=interactive",
+  sheets_gid="560419533",
+  sql_file="present_types.sql",
+  width=600,
+  height=441
+)}}
+
+This decline underscores a strategic move by developers and webmasters to adopt **Open Graph**, which originated with Facebook but has since become the **standard for social media sharing** across platforms. The **Open Graph format** offers greater flexibility and interoperability, making it the go-to choice for content optimization on Facebook as well as other social networks.
+
+{{ figure_markup(
+  image="facebook-meta-tag-usage-by-year-desktop.png",
+  caption="Facebook meta tag usage by year (desktop)",
+  description="Bar chart showing Facebook meta tag usage on desktop from 2022 to 2024. In 2024, fb:app_id appears on the highest percentage of pages at 4.9%, fb:admins on 2.4%, and fb:pages on 0.6%, with minimal or zero usage for other tags.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTE7UqINKaCzsmdVSUPZhQjlm2Az5T9wcbk4ZMZyJu-Rp8JWXtn9YRfXnAajUqQYLTCnSA2O5FEP6YF/pubchart?oid=977720399&format=interactive",
+  sheets_gid="560419533",
+  sql_file="present_types.sql",
+  width=600,
+  height=441
+)}}
+
+Despite the decreasing adoption of Facebook-specific tags, **Facebook** itself remains a key player in the **social media landscape**, with Open Graph handling most of its metadata needs. This trend reflects the consolidation of social sharing standards, where **platform-agnostic tags** provide greater reach and functionality.
+
+{{ figure_markup(
+  image="facebook-meta-tag-usage-by-device.png",
+  caption="Facebook meta tag usage by device",
+  description="Bar chart showing Facebook meta tag usage by device for desktop and mobile in 2024. The fb:app_id tag appears on the highest percentage of pages, with mobile usage at 4.9% and desktop at a similar level. The fb:admins tag is used on 2.4% of mobile pages and 0.6% on desktop, while other tags show minimal or zero usage.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTE7UqINKaCzsmdVSUPZhQjlm2Az5T9wcbk4ZMZyJu-Rp8JWXtn9YRfXnAajUqQYLTCnSA2O5FEP6YF/pubchart?oid=343986403&format=interactive",
+  sheets_gid="560419533",
+  sql_file="present_types.sql",
+  width=600,
+  height=441
+)}}
+
+## Microformats and Microformats2
+
+Microformats continue to show **limited adoption**, primarily in niche use cases where simple, semantic data is required. The `adr` tag, used for address-related data, remains the most widely adopted Microformats type, appearing on approximately **0.4% of pages** across both mobile and desktop platforms. Other tags, such as `geo` and `hReview`, have minimal usage, as more sophisticated formats like JSON-LD and Open Graph have become more prevalent.
```

Comment:
I am unsure if the following is a fair representation of microformats:

> Microformats continue to show **limited adoption**, primarily in niche use cases where simple, semantic data is required

I would say:

> Microformats continue to show limited adoption in relation to the scale of the data we analyzed. With that said, microformats have many uses, and express a rich vocabulary for marking up content (with h-entry), profiles (with h-card), HTML feeds (with h-feed), comments (with e-content), and more.

Reply:
NB: "microformats2" (with a lowercase "m") is the proper style, rather than "Microformats2".